src/steps/node.rs

```diff
@@ -276,3 +276,43 @@ pub fn deno_upgrade(ctx: &ExecutionContext) -> Result<()> {
     print_separator("Deno");
     ctx.run_type().execute(&deno).arg("upgrade").status_checked()
 }
+
+/// There is no `volta upgrade` command, so we need to upgrade each package
+pub fn run_volta_packages_upgrade(ctx: &ExecutionContext) -> Result<()> {
+    let volta = require("volta")?;
+
+    print_separator("Volta");
+
```

Comment:
We need to support the `--dry-run` option, so should check it here:

```suggestion

    if ctx.run_type().dry() {
        print_info("Updating Volta packages...");
        return Ok(());
    }
```